src/main/java/org/scalameter/japi/annotation/benchmark.java

```diff
@@ -0,0 +1,31 @@
+package org.scalameter.japi.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/** Annotates benchmark snippet.
+ *
+ *  Its value sets the scope.
```

Comment:
Could you write:

    Its value sets the scope name of the snippet.

   The following declaration:

    {{{
    @benchmark("arrays.forloops")
    public void snippetMethod(int[] xs) {
      // ...
    }
    }}}

    declares a benchmark snippet, equivalent to:

    {{{
    performance of "arrays" in {
      measure method "forloops" in {
        // ...
      }
    }
    }}}